charts/team-ns/templates/external-secrets.yaml

```diff
@@ -35,8 +35,8 @@ spec:
   data:
     {{- range .entries }}
     - key: {{ $vaultPath }}
-      name: {{ .name }}
-      property: {{ .name }}
+      name: {{ . }}
+      property: {{ . }}
```

Comment:
the more I think about it, the more I think we DO need an intermediary name for the var name, as we expect ENV vars to be UPPERCASE_SNAKE_CASE, but we don't have control over this in Vault UI. (We can't simply transform those Vault kv keys to UPPERCASE_SNAKE_CASE.)

So we do have to keep the dictionary, and make `name:` hold the UPPERCASE_SNAKE_CASE var name (should be validated as such).

Reply:
Good that we catch it now ;)

Reply:
A k8s secret can be shared among different apps. Each app can have slightly different env naming. Therefore
a user should be able to define env name at app context (if it differs from key in secret.)

We also cannot enforce people to follow UPPERCASE_SNAKE_CASE for env naming. It is up to developers and we should not dictate that convention. We provide a platform that facilitates DevOps teams not a software development framework (IMO)

Reply:
sure, if that is not standardized upon we can't force them

Reply:
Here it is!

Reply:
I see it, but we did not agree on reversing to named property. I can't find valid argument for doing it. What's the problem you are trying to solve now?